compiler/luci-interpreter/src/loader/GraphLoader.cpp

```diff
@@ -73,6 +73,25 @@ const void *getNodeData(const luci::CircleConst *node, size_t *data_size)
   }
 }
 
+const void *getNodeData(const luci::CircleCustom *node, size_t *data_size)
+{
+  if (node->custom_code() != "CircleReferencingConst")
+    return nullptr;
+
+  // helper struct which describes data loaded to custom_options of CircleReferencingConst node
+  struct ConstDataReference
+  {
+    const uint8_t *data = nullptr;
+    uint32_t size = 0;
+  };
```

Comment:
(optional) I assume this struct exist also in other file. So I think it would be better to include from header. Or add a `// TODO move to header`

Reply:
I'd prefer second option for now.